patches/minecraft/net/minecraft/entity/monster/EntityZombieVillager.java.patch

```diff
@@ -20,13 +28,36 @@
 +        this.func_190733_a(net.minecraftforge.fml.common.registry.VillagerRegistry.getId(prof));
 +    }
 +
-+    @Nullable
 +    public net.minecraftforge.fml.common.registry.VillagerRegistry.VillagerProfession getForgeProfession()
 +    {
++        if (this.prof == null)
++        {
++            this.prof = net.minecraftforge.fml.common.registry.VillagerRegistry.getById(this.func_190736_dl());
++            if (this.prof == null)
++                return net.minecraftforge.fml.common.registry.VillagerRegistry.getById(0); //Farmer
++        }
 +        return this.prof;
 +    }
 +
++    public void func_184206_a(DataParameter<?> key)
```

Comment:
In `EntityVillager`, this is annotated `@Override`, do you think that should be done here too?